AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.kt

```diff
@@ -212,7 +212,11 @@ class BasicImageFieldController : FieldControllerBase(), IFieldController {
                 // cropImage can give us more information. Not sure it is actionable so for now just log it.
                 val error: String = cropResult.error?.toString() ?: "Error info not available"
                 Timber.w(error, "cropImage threw an error")
-                CrashReportService.sendExceptionReport(error, "cropImage threw an error")
+                if ("com.canhub.cropper.CropException" in error) {
+                    Timber.i("CropException caught")
```

Comment:
I think we could at least log the exception with the Timber message, that way since it is 'i' level it will show up even in release mode builds, and if someone is affected by this they can generate a manual exception report that *should* still have the exception in it from logcat, allowing us to still investigate in depth with an affected user in case we are incorrect in the analysis that these do not matter

```suggestion
                    Timber.i("CropException caught, seemingly nothing to do", error)
```

Reply:
ok will amend it